From 43b15f8757a7f8de0340cc977ff9619741a5d43f Mon Sep 17 00:00:00 2001
From: Brian McGee <brian@bmcgee.ie>
Date: Mon, 1 Jul 2024 14:20:22 +0100
Subject: [PATCH] fix: configure toml key for global excludes

Signed-off-by: Brian McGee <brian@bmcgee.ie>
---
 config/config.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/config.go b/config/config.go
index d9e281f..be5c991 100644
--- a/config/config.go
+++ b/config/config.go
@@ -10,7 +10,7 @@ import (
 type Config struct {
 	Global struct {
 		// Excludes is an optional list of glob patterns used to exclude certain files from all formatters.
-		Excludes []string
+		Excludes []string `toml:"excludes"`
 	}
 	Formatters map[string]*Formatter `toml:"formatter"`
 }
